### DISCUSSION TITLE
When will the next version be released with changes that generate annotations on fields instead of getters for Kotlin?

### DISCUSSION THREAD

**Discussion by rachana-sudhindradani** [kobylynskyi/graphql-java-codegen#1270] [🚀 Releases]
Currently using the local build. We want to consume the latest version and proceed with our dev

**Comment by rachana-sudhindradani** [comment]
@jxnu-liguobin please update on above

**Comment by jxnu-liguobin** [comment]
cc  @kobylynskyi

**Comment by kobylynskyi** [comment]
@rachana-sudhindradani there are some pending tickets in 5.8.0 release:
https://github.com/kobylynskyi/graphql-java-codegen/milestone/47
Once those are resolved then I will release 5.8.0 that includes the changes related to Kotlin annotations

**Comment by kobylynskyi** [comment]
All tickets for 5.8.0 release are ready.
Will perform the release as soon as possible.
Thanks for the patience.

**Comment by kobylynskyi** [comment]
@rachana-sudhindradani 5.8.0 was released: https://github.com/kobylynskyi/graphql-java-codegen/releases/tag/v5.8.0
